module/version_reconfigure_test.go

```diff
@@ -0,0 +1,260 @@
+package module_test
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"go.viam.com/test"
+	goutils "go.viam.com/utils"
+	"go.viam.com/utils/pexec"
+	"go.viam.com/utils/rpc"
+
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/robot"
+	"go.viam.com/rdk/robot/client"
+	"go.viam.com/rdk/testutils"
+	"go.viam.com/rdk/utils"
+)
+
+// TODO(pre-merge) refactor this somewhat-shared function into a test_helpers esque file.
```

Comment:
I assume these comments are no longer relevant? (There's one on `modifyCfg`, too).

Reply:
Err yeah. Let me clean up some documentation in the test.